spec/chromium-spec.js

```diff
@@ -371,7 +371,7 @@ describe('chromium feature', () => {
 
     it('disables node integration when it is disabled on the parent window', (done) => {
       let b = null
-      listener = (event) => {
+      listener = async (event) => {
```

Comment:
Why did this become `async`?

Reply:
oops, i changed it because i had a wait loop in here, but then i removed the wait loop. fixed.